src/pybind/mgr/rook/module.py

```diff
@@ -312,32 +314,22 @@ def get_inventory(self, node_filter=None, refresh=False):
         for node_name, node_devs in devs.items():
             devs = []
             for d in node_devs:
-                dev = orchestrator.InventoryDevice()
-
-                # XXX CAUTION!  https://github.com/rook/rook/issues/1716
-                # Passing this through for the sake of completeness but it
-                # is not trustworthy!
-                dev.blank = d['empty']
-                dev.type = 'hdd' if d['rotational'] else 'ssd'
-                dev.id = d['name']
-                dev.size = d['size']
-
-                if d['filesystem'] == "" and not d['rotational']:
-                    # Empty or partitioned SSD
-                    partitioned_space = sum(
-                        [p['size'] for p in d['Partitions']])
-                    dev.metadata_space_free = max(0, d[
-                        'size'] - partitioned_space)
-
+                dev = inventory.Device(
+                    path=d['name'],
+                    sys_api=dict(
+                        rotational=str(d['rotational']),
```

Comment:
Seems `d['rotational']` is in `bool` type, so this one will be converted to `"True"` or `"False"`. `sys_api['rotational']` is supposed to be `"1"` or `"0"`.